src/metaTypes.h

```diff
@@ -35,18 +36,18 @@ namespace METAIO_NAMESPACE {
 typedef char                MET_ASCII_CHAR_TYPE;
 typedef char                MET_CHAR_TYPE;
```

Comment:
Good idea. After reading what was said on the ITK mailing list, I agree that (u)int8_t should be used for (unsigned) char to avoid signed/unsigned issues (more specifically to be sure that char is signed since char definition is platform dependent). I will address this issue.